docs/concepts/what-is-a-cloud-namespace-name.md

```diff
@@ -13,7 +13,7 @@ It cannot be changed after the Namespace is provisioned.
 
 Each Namespace Name must conform to the DNS naming rules:
 
-- A Namespace Name must contain at least 1 character and no more than 34 characters.
+- A Namespace Name must contain at least 2 characters and no more than 34 characters.
 - A Namespace Name must begin or end with a letter or number and can contain only letters, numbers, and the hyphen (-) character.
 - Each hyphen (-) character must be immediately preceded _and_ followed by a letter or number; consecutive hyphens are not permitted.
```

Comment:
good point on consecutive hyphens but we currently don't reject them...

Reply:
OK as is for now, or should I do one of the following?

- Remove the list item.
- Change sentence to "For forward compatibility, each hyphen (-) character should be immediately preceded _and_ followed by a letter or number; consecutive hyphens should not be used."

@mastermanu @feihuang

Reply:
I think we should just keep it as it is and on backend we'll just enforce no consecutive dashes before GA.

On a minor point, line 14 talks about DNS naming rules but since 1 character is valid dns name, our rules have deviated from dns rules?

Reply:
Good point. I'll modify line 14 to skip the reference to DNS.